.github/workflows/pr-extra.yml

```diff
@@ -0,0 +1,19 @@
+name: Extra
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+jobs:
+  vulns:
+    name: Vulnerability scanner
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+      # We cannot use nancy-github-action because it is outdated, so it's better to use the latest
+      # docker image for the validation
+      - name: nancy
+        run: go list -m all | docker run -i sonatypecommunity/nancy:latest
```

Comment:
You should rather use:
```
go list -json -m all | nancy
```
(see [deprecation notice](https://github.com/sonatype-nexus-community/nancy#what-is-the-best-usage-of-nancy))

Reply:
thanks I didn't know

Reply:
done